ibm-cloud-pak-for-data/azuredeploy.parameters.json

```diff
@@ -30,13 +30,13 @@
 			"value": "GEN-AZUREAD-AKS-APPID-SECRET"
 		},
 		"rhelUsername": {
-			"reference": "GEN-KEYVAULT-CLOUDPAK-USER-REFERENCE"
+			"value": "ignoreme"
```

Comment:
These param values were added specifically for this sample, why are we removing them?

Reply:
@bmoore-msft the following three keys are not used now as we are deploying OpenShift origin (free version) for test pipeline. The earlier keys have expired and to avoid expiration issue in future we are doing this change. You can clean these up in the keyvault. thanks.
`GEN-KEYVAULT-CLOUDPAK-USER-REFERENCE`
`GEN-KEYVAULT-CLOUDPAK-PASSWORD-REFERENCE`
`GEN-KEYVAULT-CLOUDPAK-POOL-REFERENCE`

Reply:
Ok, then we need to remove the params from the parameter file and set defaultValues in the template (e.g. empty string)

Reply:
what about these?

`GEN-KEYVAULT-CLOUDPAK-APIUSER-REFERENCE`
`GEN-KEYVAULT-CLOUDPAK-APIKEY-REFERENCE`

Reply:
@bmoore-msft  the above two keys are required. please retain them.

Reply:
> Ok, then we need to remove the params from the parameter file and set defaultValues in the template (e.g. empty string)

@bmoore-msft moved the params to the template.